src/App.vue

```diff
@@ -160,7 +157,9 @@ export default {
           .map(o => o.trim())
 
         this.originRegExps = origins.map(o => {
-          const str = `^${escapeRegexp(o).replace(/\\\*/g, '.*')}$`
+          if (o === '*') return /^.*$/
+
+          const str = `^${escapeRegexp(o).replace(/\\\*/g, '[^/]*')}$`
```

Comment:
Did you try to skip escapeRegexp as I suggested with the following:

```js
// escaping is not needed here
return new RegExp(`^${o.replace(/\*/g, '[^/]*')}$`)
```

Reply:
Yes I tried to remove the escaping from escape-string-regex.
Unfortunately, the generated regexps are incorrect.

For example, `https://example.com:3210` are transformed into `/^https://example.com:3210$/` which doesn't even match the origin `https://example.com:3210` because `/` and real `.` aren't escaped.

Reply:
Right.